Core.DomainModel/ItSystemUsage/ItSystemUsage.cs

```diff
@@ -550,28 +550,29 @@ public Maybe<OperationError> UpdateSystemVersion(string version)
 
         public void ResetUserCount()
         {
-            UserCount = UserCount.BELOWTEN;
+            UserCount = null;
         }
 
-        public Maybe<OperationError> SetExpectedUsersInterval((int lower, int? upperBound) newIntervalValue)
+        public Maybe<OperationError> SetExpectedUsersInterval((int? lower, int? upperBound) newIntervalValue)
         {
             switch (newIntervalValue)
             {
                 case (0, 9):
-                    UserCount = UserCount.BELOWTEN;
+                    UserCount = DomainModel.ItSystem.DataTypes.UserCount.BELOWTEN;
                     break;
                 case (10, 50):
-                    UserCount = UserCount.TENTOFIFTY;
+                    UserCount = DomainModel.ItSystem.DataTypes.UserCount.TENTOFIFTY;
                     break;
                 case (50, 100):
-                    UserCount = UserCount.FIFTYTOHUNDRED;
+                    UserCount = DomainModel.ItSystem.DataTypes.UserCount.FIFTYTOHUNDRED;
                     break;
                 case (100, null):
                 case (100, int.MaxValue):
-                    UserCount = UserCount.HUNDREDPLUS;
+                    UserCount = DomainModel.ItSystem.DataTypes.UserCount.HUNDREDPLUS;
                     break;
                 default:
-                    return new OperationError("Invalid user count. Please refer to input documentation", OperationFailure.BadInput);
+                    UserCount = DomainModel.ItSystem.DataTypes.UserCount.UNDECIDED;
+                    break;
```

Comment:
I'm not sure if this switch is correct, or should the 'default' still return an error, and there should be a "null" case